ui/components/component-library/picker-network/picker-network.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import README from './README.mdx';
+import { PickerNetwork } from './picker-network';
+
+export default {
+  title: 'Components/ComponentLibrary/PickerNetwork',
+  id: __filename,
+  component: PickerNetwork,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    label: {
+      control: 'text',
+    },
+    src: {
+      control: 'text',
+    },
+  },
+  args: {
+    label: 'Imported',
```

Comment:
Maybe a more appropriate label for the example would be. We should try to show the component as it would be implemented. 

```suggestion
    label: 'Avalanche C-Chain',
```